tests/python/gaia-ui-tests/gaiatest/tests/unit/test_override_defaults.py

```diff
@@ -5,7 +5,9 @@
 from gaiatest import GaiaTestCase
 
 TEST_DEFAULTS = {'ftu.manifestURL': 'app://ftu.gaiamobile.org/manifest.webapp'}
-
+TEST_DEFAULT_PREFS = {'dom.requestSync.minInterval': 99,
+                      'ril.debugging.enabled': True,
+                      'b2g.neterror.url': 'blahblah'}
```

Comment:
Oh, ok. I assumed I had to use real prefs, because the ftu.manifestURL setting also exists.